assignment-1/Team.java

```diff
@@ -0,0 +1,27 @@
+import java.util.ArrayList;
+
+public class Team extends Players {
+    public String Name;
```

Comment:
1. never declare instance variables as public, always make their private and use getter methods
2. follow the proper naming convention, variable, and method names must begin with a lowercase character.